.github/workflows/major-version-upgrade.yml

```diff
@@ -6,8 +6,8 @@ jobs:
     env:
       OLD_INSTALL_DIR: psql_source
       NEW_INSTALL_DIR: psql_target
-      ENGINE_BRANCH_FROM: BABEL_2_X_DEV__PG_14_X
-      EXTENSION_BRANCH_FROM: BABEL_2_X_DEV
+      ENGINE_BRANCH_FROM: oid-rework-14-x-mvu
```

Comment:
this is for empty database only. Main MVU tests are driven form config while which specified source and target versions.